packages/api/src/beacon/server/lightclient.ts

```diff
@@ -24,5 +25,53 @@ export function getRoutes(config: IChainForkConfig, api: Api): ServerRoutes<Api,
         return Buffer.from(serializeProof(proof));
       },
     },
+    getBootstrap: {
+      ...serverRoutes.getBootstrap,
```

Comment:
tried de-duplicating some of the code here, but was a bit tricky getting the types to line up right. If this is a blocker, then I'll revisit.